Code/GraphMol/MolEnumerator/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+remove_definitions(-DRDKIT_GRAPHMOL_BUILD)
+add_definitions(-DRDKIT_MOLENUMERATOR_BUILD)
+rdkit_library(MolEnumerator
+              MolEnumerator.cpp
+              LINK_LIBRARIES FileParsers ChemReactions ChemTransforms 
+              SubstructMatch GraphMol RDGeneral)
```

Comment:
We don't need all those yet, right? Maybe thinking ahead for link atoms and SRUs?

Reply:
yeah, exactly.